app/views/users/_show.html.haml

```diff
@@ -44,10 +44,20 @@
   .col-sm-8{:style => "background-color:white"}
     %h1
       = @user.name.blank? ? @user.login : @user.name
-      - if @user.is_admin? || @user.is_curator?
-        %small
-          = @user.is_admin? ? t(:admin) : t(:curator)
-          = link_to help_path(:anchor => "curators") do
+      - admin_site = @user.site_admins.live.first.try(:site)
+      - if @user.is_admin? || @user.is_curator? || admin_site
+        %small.role
+          - info_link = help_path(anchor: "curators")
+          - if @user.is_admin?
+            = t(:site_staff, site_name: "iNaturalist")
+          - elsif admin_site
+            - site_link = link_to( admin_site.name, admin_site.url, class: "site")
+            = raw t(:site_site_admin, site_name: site_link)
+            = " (#{t("places_name.#{admin_site.place.name.parameterize.underscore}")})"
```

Comment:
@pleary not all sites have a place (e.g. the default site), so theoretically if someone was a site-admin but not staff for a site without a place this would raise an error, right? It's doing that for me locally but my local sites are definitely a bit weirder than they are in prod, so maybe this will just never happen.

Reply:
Thanks for catching that. I added a conditional to only render the place name if we have one